<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_MainWindow</class>
 <widget class="QMainWindow" name="_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTK2CSV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>RTK: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gpxkmlInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="gpxkmlButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>IMG: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imgInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="imgButton">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="text">
         <string>Converter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
